doc/news/changes/minor/20180408LucaHeltai

```diff
@@ -0,0 +1,5 @@
+New: The Manifold class now requires the implementation of a Manifold::clone() method in
```

Comment:
This should be in the `incompatibilities` section.

Reply:
It is also there.

Reply:
In a different PR? Maybe I am just overlooking it, but I can't find it here.

Reply:
Sorry, you were right. It is now in the `incompatibilities` section.